chargebee-js-vue/src/components/Provider.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <fragment v-if="validated">
```

Comment:
Fragment inserts a div component, this can conflict with the styles implemented on the by the merchant on the wrapper component. Can we try to use render function and return the slots directly based on the validation?